my_project/utils.py

```diff
@@ -273,6 +273,44 @@ def title_with_tooltip(text, tooltip_text, id_button):
         ],
     )
 
+def title_with_link(text, tooltip_text, id_button,doc_link):
```

Comment:
change:
tooltip_text = "Click to access the official documentation" 

so you do not need to pass this everytime you call this function